src/components/Modals/CreateGnosisSafeModal/index.tsx

```diff
@@ -55,7 +55,7 @@ const CreateGnosisSafeModalContent: FunctionComponent<{
 			if (newGnosis) {
 				setStage("create")
 			} else {
-				console.log("TODO")
+				console.log("TODO", stage)
```

Comment:
Why do you need this `stage` being consoled? I personally don't like even the fact that we're having this `console.log`s with "TODO" message, but at least it doesn't expose any code internals, while this `stage` thingy definitely something deep in our code.